src/styles/menu.scss

```diff
@@ -57,11 +57,12 @@
       }
 
       &--subcategory {
-        font-size: 0.9rem;
+        font-size: 0.875rem;
```

Comment:
nit: we can use `--amplify-font-sizes-small` token here for the same size.

Reply:
Ah thank you so much, I completely missed it when checking the font size variables